tests/output/ConsoleInteractionTest.py

```diff
@@ -1082,3 +1082,25 @@ def test_source_lines(self):
                 {abspath('file'): ('def fun():\n', '    pass  \n')})
             self.assertEqual(stdout.getvalue(),
                              "('    pass  \\n',)\n")
+
+    def test_show_bears_specified_in_args(self):
+        from coalib import coala
+        from tests.TestUtilities import bear_test_module, execute_coala
+        with bear_test_module():
+            retval, stdout, stderr = execute_coala(
+                coala.main, 'coala', '-B', '--bears',
+                'JavaTestBear')
+            self.assertEqual(retval, 0)
+            # 1 bear plus 1 line holding the closing colour escape sequence.
+            self.assertEqual(len(stdout.strip().splitlines()), 2)
+
+    def test_show_bears_specified_in_args_regex(self):
+        from coalib import coala
+        from tests.TestUtilities import bear_test_module, execute_coala
+        with bear_test_module():
+            retval, stdout, stderr = execute_coala(
+                coala.main, 'coala', '-B', '--bears',
+                '*Java*', '*Space*')
+            self.assertEqual(retval, 0)
+            # 2 bear plus 1 line holding the closing colour escape sequence.
+            self.assertEqual(len(stdout.strip().splitlines()), 3)
```

Comment:
I would like to see if we test for the output strings, to see whether the correct bears were matched. If you worry about color, enable `--no-color` in arguments (I hope that's the right flag)